wagtail/wagtailadmin/static_src/wagtailadmin/scss/layouts/page-editor.scss

```diff
@@ -348,9 +348,26 @@
 }
 
 // Footer control bar for perfoming actions on the page
+@media screen and (max-width: $breakpoint-mobile) {
+    footer .actions {
+        width: 100%;
+    }
+
+    footer .meta p {
+        white-space: normal;
+    }
+}
+
```

Comment:
Could we move these `@media` `.actions` and `.meta` changes into `core.scss` since the base for this particular is located in there? That way we can consolidate it into its origin, rather than overriding it here specifically.